types/index.d.ts

```diff
@@ -69,6 +70,49 @@ export interface QuillOptionsStatic {
     bounds?: HTMLElement | string;
     scrollingContainer?: HTMLElement | string;
     strict?: boolean;
+    /**
+     * Registry is a data structure provided by the Parchment library. Registry maps a node type to its
+     * corresponding style blot. It contains a register method to allow users to customize the blot map.
+     * Quill uses a registry to know how to translate between html contents and Delta operations. Delta
+     * is a library that Quill uses to represent the editor’s contents and changes. By default, Quill uses
+     * a global registry. In the versions below Quill 2.0, you're supposed to configure formats using the
+     * formats option. However, this API is changed in Quill 2.0+. Formats option does not fully prevent
+     * Quill from rendering pasted formatted contents that are not supported. In the 2.0+ version of Quill
+     * that we are using, there is an undocumented way to provide Quill with a custom registry. Doing so
+     * allows Quill to strip unsupported styles when contents are being pasted into an editor.
```

Comment:
> allows Quill to strip unsupported styles

Nitpick: I'd say it prevents Quill from rendering unwanted styles rather than allows it to strip them.